### PR TITLE
Fix 3rd att transmission status

### PR DESCRIPTION
**Pull request by cristinasewell**
## Description
Have corrected the transmission status for the 3rd harmonic by reading this value from the `readback_3rd` which is the signal for `:COM:R3_CUR` this PV - based on a little bit of research this seems the appropriate PV to use for the transmission of the 3rd harmonic. Please see the discussion here: https://github.com/pcdshub/pcdsdevices/issues/675#issuecomment-755131529

## Motivation and Context

Fix the status information for the 3rd harmonic transmission value.
## How Has This Been Tested?

```
happi load mfx_attenuator

In [1]: mfx_attenuator
Out[1]:
filter # |0|1|2|3|4|5|6|7|8|9|
 OUT     |X|X| | | |X|X|X|X|X|
 IN      | | |X|X|X| | | | | |
Transmission for 1st harmonic (E=13.000 keV): 4.0911E-01
Transmission for 3rd harmonic (E=39.000 keV): 9.2504E-01
```

```
happi load at1l0

In [1]: at1l0
Out[1]:
filter # |0|1|2|3|4|5|6|7|8|
 OUT     |X|X|X|X|X|X| | | |
 IN      | | | | | | |X|X|X|
Transmission for 1st harmonic (E=15.862 keV): 1.0000E+00
```
## Where Has This Been Documented?

N/A

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [ ] Test suite passes locally
- [ ] Test suite passes on travis
- [ ] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [ ] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/742-Fix_3rd_Att_Transmission_Status.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- The transmission status value for the 3rd harmonic has been fixed, it was previously using the wrong value.
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/742-Fix_3rd_Att_Transmission_Status.rst:30]:
```suggestion
- cristinasewell
```

**Review by ZLLentz — APPROVED** [review]
I think this looks good

**Comment by klauer** [comment]
This looks good but will create an immediate conflict for #740 (https://github.com/pcdshub/pcdsdevices/pull/740/files#diff-b465666f0f2a2446a134e2d3c65712581498d26afff1285facac28002b4c52acL319). I touched that line but looks like I neglected to really fix the status message.

Sorry for the extra work, but I'd think merging #740 first and then applying an appropriate fix on top would be better order-wise.

**Comment by cristinasewell** [comment]
ohh I did not see you have that - yeah that sounds good, this is an easy fix anyway - probably can be done in your branch directly if you're still working on it

**Comment by klauer** [comment]
Thanks @cristinasewell - the other PR was merged if you're ready to fix this up.

**Review by klauer — APPROVED** [review]
Thanks - and sorry for getting in the way of the original PR.

**Comment by cristinasewell** [comment]
I need to double check the Energy conversion before merging, the energy numbers look a bit too large I think

**Comment by klauer** [comment]
The scaling may be off - `1e-3` instead of `1e3`?

File: docs/source/upcoming_release_notes/742-Fix_3rd_Att_Transmission_Status.rst (hunk 1 above)
Comment (cristinasewell) [reply docs/source/upcoming_release_notes/742-Fix_3rd_Att_Transmission_Status.rst]: Thanks Zach I forgot about this - changed this in 2d6d7ad

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**